scripts/archive-subscribers.sh

```diff
@@ -6,25 +7,31 @@ git fetch origin
 git pull origin master
 
 # Set up password for Mailman
-MAILMANPASSDIR=${MAILMANPASSFILE:-"${HOME}/GitConfig/InitFiles/GTALUG/"}
+MAILMANPASSDIR=${MAILMANPASSFILE:-${HOME}/GitConfig/InitFiles/GTALUG/}
```

Comment:
The bash doc I was looking at didn't have quotes and functionally they make no difference.